stories/control/line.stories.tsx

```diff
@@ -1,7 +1,15 @@
 import { Meta } from '@storybook/react';
 import React from 'react';
 
-import { Axis, Legend, LineSeries, LineSeriesProps, Plot } from '../../src';
+import {
+  Axis,
+  Legend,
+  LineSeries,
+  LineSeriesProps,
+  Plot,
+  BarSeries,
+} from '../../src';
+import { DEFAULT_CONFIG } from '../utils';
 
 export default {
   title: 'API/LineSeries',
```

Comment:
The bar series probably deserves it's own title even if it shares the same API for now

Reply:
It should have its own story file too